platforms/ubuntutouch/manifest.json.in

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "org.kde.gcompris",
+    "description": "GCompris",
+    "architecture": "@CLICK_ARCH@",
+    "title": "gcompris",
+    "hooks": {
+        "gcompris": {
+            "apparmor": "gcompris.apparmor",
+            "desktop":  "gcompris.desktop"
+        }
+    },
+    "version": "1.0.1",
+    "maintainer": "lionel <lionel.duboeuf@gmail.com>",
```

Comment:
Just to be sure, do you want this email here or lduboeuf@ouvaton.org?

Reply:
thanks ;-)